server/src/main/scala/services/exports/flights/templates/CedatFlightsExport.scala

```diff
@@ -70,7 +71,7 @@ case class CedatFlightsExport(start: SDateLike, end: SDateLike, terminal: Termin
     ) ++ List(fws.apiFlight.bestPcpPaxEstimate.toString) ++ splitsForSources
   }
 
-  override def rowValues(fws: ApiFlightWithSplits): Seq[String] = (flightWithSplitsToCsvRow(fws) :::
+  override def rowValues(fws: ApiFlightWithSplits, maybeManifest: Option[VoyageManifest]): Seq[String] = (flightWithSplitsToCsvRow(fws) :::
```

Comment:
maybeManifest parameter is not used in rowValues .

Reply:
It implements a flights export trait which is used for all exports. The other exports do use the manifests, but the Cedat one here has to stay unchanged so we don't break their systems that rely on our arrivals export